Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```diff
@@ -172,74 +163,104 @@ script:
     - contextPath: CylanceProtect.Device.Name
       description: Device name.
       type: string
+  - arguments:
+    - default: true
+      description: The hostname (DNS name)
+      name: hostname
+      required: true
+    description: Allows a caller to request a specific device resource belonging to
+      a Tenant by hostname
+    name: cylance-protect-get-device-by-hostname
+    outputs:
+    - contextPath: CylanceProtect.Device.AgentVersion
+      description: The CylancePROTECT Agent version installed on the device.
+      type: date
+    - contextPath: CylanceProtect.Device.DateFirstRegistered
+      description: The date and time (in UTC) when the device record was created.
+      type: date
+    - contextPath: CylanceProtect.Device.BackgroundDetection
+      description: If true, the Agent is currently running.
+      type: boolean
+    - contextPath: CylanceProtect.Device.DateLastModified
+      description: The date and time (in UTC) when the device record was last modified.
+      type: date
+    - contextPath: CylanceProtect.Device.DateOffline
+      description: The date and time (in UTC) when the device last communicated with
+        the Console.
+      type: date
+    - contextPath: CylanceProtect.Device.Hostname
+      description: The hostname for the device.
+      type: string
+    - contextPath: CylanceProtect.Device.ID
+      description: The unique identifier for the device.
+      type: string
+    - contextPath: CylanceProtect.Device.IPAddress
+      description: The list of IP addresses for the device.
+      type: Unknown
+    - contextPath: CylanceProtect.Device.MACAddress
+      description: The list of MAC addresses for the device.
+      type: Unknown
+    - contextPath: CylanceProtect.Device.IsSafe
+      description: If true, there are no outstanding threats.
+      type: boolean
+    - contextPath: CylanceProtect.Device.UpdateAvailable
+      description: If true, there is available update for the device.
+      type: boolean
+    - contextPath: CylanceProtect.Device.State
+      description: Machine state.
+      type: string
+    - contextPath: Endpoint.Hostname
+      description: Device hostname.
+      type: string
+    - contextPath: Endpoint.MACAddress
+      description: The list of MAC addresses for the device.
+      type: Unknown
+    - contextPath: Endpoint.IPAddress
+      description: The list of IP addresses for the device.
+      type: Unknown
+    - contextPath: Endpoint.OSVersion
+      description: Device OS version.
+      type: string
+    - contextPath: CylanceProtect.Device.OSVersion
+      description: Device OS version.
+      type: string
+    - contextPath: CylanceProtect.Device.Name
+      description: Device name.
+      type: string
```

Comment:
Seems you had AgentVersion output which was not really outputted. Also you haven't added Policy outputs to Device. I have added them here, double check to see if anything is missing.
Also, added TODO for what I added for description and type. Please fill
adding a suggestion, let me know if I missed something and it was configured correctly
```suggestion
    - contextPath: CylanceProtect.Device.DateFirstRegistered
      description: The date and time (in UTC) when the device record was created.
      type: date
    - contextPath: CylanceProtect.Device.BackgroundDetection
      description: If true, the Agent is currently running.
      type: boolean
    - contextPath: CylanceProtect.Device.DateLastModified
      description: The date and time (in UTC) when the device record was last modified.
      type: date
    - contextPath: CylanceProtect.Device.DateOffline
      description: The date and time (in UTC) when the device last communicated with
        the Console.
      type: date
    - contextPath: CylanceProtect.Device.Hostname
      description: The hostname for the device.
      type: string
    - contextPath: CylanceProtect.Device.ID
      description: The unique identifier for the device.
      type: string
    - contextPath: CylanceProtect.Device.IPAddress
      description: The list of IP addresses for the device.
      type: Unknown
    - contextPath: CylanceProtect.Device.MACAddress
      description: The list of MAC addresses for the device.
      type: Unknown
    - contextPath: CylanceProtect.Device.IsSafe
      description: If true, there are no outstanding threats.
      type: boolean
    - contextPath: CylanceProtect.Device.UpdateAvailable
      description: If true, there is available update for the device.
      type: boolean
    - contextPath: CylanceProtect.Device.State
      description: Machine state.
      type: string
    - contextPath: Endpoint.Hostname
      description: Device hostname.
      type: string
    - contextPath: Endpoint.MACAddress
      description: The list of MAC addresses for the device.
      type: Unknown
    - contextPath: Endpoint.IPAddress
      description: The list of IP addresses for the device.
      type: Unknown
    - contextPath: Endpoint.OSVersion
      description: Device OS version.
      type: string
    - contextPath: CylanceProtect.Device.OSVersion
      description: Device OS version.
      type: string
    - contextPath: CylanceProtect.Device.Name
      description: Device name.
      type: string
    - contextPath: CylanceProtect.Device. LastLoggedInUser
      description: TODO add description.
      type: TODO  add the type
    - contextPath: CylanceProtect.Device.UpdateType
      description: TODO add description.
      type: TODO  add the type
    - contextPath: CylanceProtect.Device.Policy.ID
      description: TODO add description.
      type: TODO  add the type
    - contextPath: CylanceProtect.Device.Policy.Name
      description: TODO add description.
      type: TODO  add the type
```